tin/templates/base.html

```diff
@@ -76,10 +76,6 @@
           {% endif %}
         {% endif %}
       {% endif %}
-      {% if docs_app %}
-        <li><i class="fa fa-chevron-right"></i></li>
-        <li><a href="{% url 'docs:index' %}">Docs</a></li>
```

Comment:
This line should probably not be deleted on second thought - putting this here to remind myself.